<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyEditorDialog</class>
 <widget class="QDialog" name="StudyEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a new study</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QGroupBox" name="study_properties_groupBox">
     <property name="title">
      <string>Study Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="studyname_label">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="studyname_lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="outputdir_label">
              <property name="text">
               <string>Output directory:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="outputdir_lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="outputdir_button">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../../resources/morphologist.qrc">
                <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="backup_filename_label">
              <property name="text">
               <string>Backup file:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="backup_filename_lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="backup_filename_button">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../../resources/morphologist.qrc">
                <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="link_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>39</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../resources/morphologist.qrc">
            <normaloff>:/icons/link.png</normaloff>
            <normalon>:/icons/unlink.png</normalon>:/icons/link.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>7</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="parameter_template_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Output directory organization:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="parameter_template_combobox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableView" name="subjects_tableview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="add_subjects_groupBox">
          <property name="title">
           <string>Add subjects</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="add_one_subject_from_a_file_button">
             <property name="text">
              <string>From files</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="add_subjects_from_a_study_directory_button">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>From a study directory</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="add_subjects_from_database_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>From Brainomics database</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="add_subjects_from_a_pattern_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>All files following a pattern</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="edit_subjects_groupBox">
          <property name="title">
           <string>Edit / Remove subjects</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="edit_subjects_name_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Edit subjects name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="edit_subjects_group_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Edit subjects group</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove_subjects_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove subjects</string>
             </property>
             <property name="shortcut">
              <string>Del</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>242</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="apply_cancel_buttons">
          <property name="standardButtons">
           <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/morphologist.qrc"/>
 </resources>
 <connections/>
</ui>
